hack/tools/go.mod

```diff
@@ -10,10 +10,10 @@ require (
 	github.com/joelanford/go-apidiff v0.4.0
 	github.com/onsi/ginkgo v1.16.5
 	gotest.tools/gotestsum v1.8.2
-	k8s.io/code-generator v0.24.4
-	sigs.k8s.io/cluster-api/hack/tools v0.0.0-20220912130527-3c462340b297
+	k8s.io/code-generator v0.25.0
+	sigs.k8s.io/cluster-api/hack/tools v0.0.0-20221008002318-c9b4a0f0fc38
 	sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20211110210527-619e6b92dab9
-	sigs.k8s.io/controller-tools v0.9.2
+	sigs.k8s.io/controller-tools v0.10.0
```

Comment:
and why this one too?

Reply:
auto-updated via `sigs.k8s.io/cluster-api/hack/tools` upgrade